app/controllers/application_controller.rb

```diff
@@ -9,6 +9,7 @@ def authenticate!
     return if logged_in? || user_from_token_authentication
 
     respond_to do |format|
+      format.any { redirect_to root_path }
```

Comment:
Question: what takes precedence after adding `format.any`?

Reply:
We could also add a custom MIME type to respond to PDF format, ultimately referencing as `format.pdf { redirect_to root_path }`. From what I looked at, this could be achieved by adding the following in `config/initializers/mime_types.rb`:
```ruby
Mime::Type.register "application/pdf", :pdf
```
Would that also work? I'm trying to reduce what the scope here to be more specific.

Reply:
Yes, I think that would work, well noted!
I added it because I got an exception when I tried go back from the "pdf" page with an expired session.
You're right, maybe it is too generic to have `format.any`, I'll test it.